backend/tests/test_partners.py

```diff
@@ -221,18 +220,22 @@ def test_partner_pagination(client, example_partners, access_token):
     assert res.status_code == 404
 
 
-def test_add_member_to_partner(db_session, example_members):
-    created = example_members["publisher"]
+# def test_add_member_to_partner(db_session, example_members):
+    # created = example_members["publisher"]
 
-    partner_member_obj = (
-        db_session.query(PartnerMember)
-        .filter(PartnerMember.id == created["id"])
-        .first()
-    )
+    # partner_member_obj = (
+    #     db_session.query(PartnerMember)
+    #     .filter(PartnerMember.id == created["id"])
+    #     .first()
+    # )
 
-    assert partner_member_obj.partner_id == created["partner_id"]
-    assert partner_member_obj.user_id == created["user_id"]
-    assert partner_member_obj.role == created["role"]
+    # assert partner_member_obj.partner_id == created["partner_id"]
+    # assert partner_member_obj.email == created["email"]
+    # assert partner_member_obj.role == created["role"]
+    """
```

Comment:
No need to worry about testing the mailing functionality here. We can handle that in a seperate test. For the purposes of these tests, just assume the mail feature is working.